astropy/units/tests/test_quantity_array_methods.py

```diff
@@ -468,13 +481,6 @@ def test_item(self):
         with pytest.raises(ValueError):
             q1.itemset()
 
-        q1[1] = 1
```

Comment:
This removal seems unrelated to me. Why is this deleted?

Reply:
It is still there at l.464 - basically, I split the test in two parts, one that just checks `.item()` and one that tests `.itemset` - the git diff is really confusing here!

Reply:
Ah, I see now. Thanks for the clarification!